src/System.Buffers/ref/System.Buffers.builds

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <Project Include="System.Buffers.csproj">
+      <TargetGroup>netstandard1.1</TargetGroup>
```

Comment:
Remove the targetgroup.  We keep a "default" build, AKA TargetGroup='' so that we have something useful when we do `msbuild` from the commandline.